### ISSUE TITLE
[BUG] SEN data seems to have some incorrect values - CF greater than one for a number of techs.

### ISSUE THREAD

**Issue by tniet** [OSeMOSYS/osemosys_global#179]
### Conda environment check

- [X] I have tried using the `osemosys-global` conda environment.

### Current Behavior

When running Senegal I had a quick look through the data file and noticed that there were a number of places where the CF for CSP was higher than 1 in many places.

### Steps To Reproduce

Run OG with SEN as the country.

### Log output

```shell
param default 1 : CapacityFactor :=
...
GLOBAL PWRHYDSENXX01 S2D2 2049 0.5294
...
GLOBAL PWRHYDSENXX01 S4D5 2050 0.2418
GLOBAL PWRCSPSENXX01 S1D1 2020 0
...
GLOBAL PWRCSPSENXX01 S1D1 2021 0
...
GLOBAL PWRCSPSENXX01 S1D2 2050 0.2605
GLOBAL PWRCSPSENXX01 S1D3 2020 1.646
GLOBAL PWRCSPSENXX01 S1D3 2021 1.646
GLOBAL PWRCSPSENXX01 S1D3 2022 1.646
...
GLOBAL PWRCSPSENXX01 S1D4 2031 1.1404
...
GLOBAL PWRCSPSENXX01 S1D4 2049 1.1404
GLOBAL PWRCSPSENXX01 S1D4 2050 1.1404
GLOBAL PWRCSPSENXX01 S1D5 2020 0.0161
GLOBAL PWRCSPSENXX01 S1D5 2021 0.0161
...
```

**Comment by maartenbrinkerink** [comment]
@tniet I don't think we ever made the modelling of CSP functional for OG. The CF timeseries based on the PLEXOS-World dataset are not useable in a context where CSP storage is not modelled separately (i.e. any surplus CF for CSP goes into thermal storage). We kind of ignored it since it had neglectable impact when looking at 2015 installed capacities. Might be different ofcourse for future capacities (especially for specific countries). Happy to discuss how the inclusion of CSP can be tackled but don't think it is a high priority for the coming months imo. Options could be;

-Using daily CF's with the assumption that thermal storage during night time is big enough to support generation at max capacity.
-Modelling CSP storage as individual technologies linked to a CSP generator technology (not sure if this can be done in OSeMOSYS)
-Excluding CSP from the modelling
